### PR TITLE
Completion of SSE-KMS doc rewrite

### PR DESCRIPTION
**Pull request by ravindk89**
# Summary

With the addition of the updated Conditional logic, we can significantly streamline the conversion process.

Initial PR commit is to show how the conditionals work w/ both in-page conditional headers and included conditional headers.

Further work in this PR will be on the remaining KMS solutions.

### CODE REVIEW

**Review by djwfyi — APPROVED** [review]
LGTM

File: source/includes/container/common-minio-kes.rst [hunk 1]
```diff
@@ -69,33 +69,27 @@ The following commands create two TLS certificates that expire within 30 days of
     :class: copyable
     :substitutions:
 
+    # These commands output keys to |path|/certs on the host operating system
```
Comment (djwfyi) [inline source/includes/container/common-minio-kes.rst:72]: `|path|` needs to be corrected. It is not replacing correctly in the rendered file.

**Review by djwfyi — APPROVED** [review]
One replacement that needs to be corrected.

Consider updating code blocks that have `\` in multi-line to just two spaces and then the `\`. The replacements are causing havoc with trying to line them up.

**Comment by djwfyi** [comment]
Approved / LGTM with one fix and an optional suggestion in my review.

File: source/includes/container/common-minio-kes.rst (hunk 1 above)
Comment (ravindk89) [reply source/includes/container/common-minio-kes.rst:72]: Done

Also: `\s*\\` <- regex to replace all of those kinds of code-block style things. Fun.

**Pull request merged**